__tests__/app.test.js

```diff
@@ -64,5 +64,7 @@ describe("GET /api/articles/:article_id", () => {
         );
       });
   });
+  it("should return with a 404 - not found error if an invalid id is given", () => {
+    return request(app).get("/api/articles/0").expect(404);
+  });
```

Comment:
Please take another look at the error-handling notes. You've gotten your wires a little crossed here. There should be separate tests for 400: invalid id (e.g, where a string is passed instead of a number) and 404: id not found (the id is of the correct type, but doesn't exist in the database)